src/main/java/io/appium/java_client/InteractsWithApps.java

```diff
@@ -171,11 +170,10 @@ default boolean terminateApp(String bundleId) {
      * @return true if the app was running before and has been successfully stopped.
      */
     default boolean terminateApp(String bundleId, @Nullable BaseTerminateApplicationOptions options) {
-        String[] parameters = options == null ? new String[]{"bundleId"} :
-                new String[]{"bundleId", "options"};
-        Object[] values = options == null ? new Object[]{bundleId} :
-                new Object[]{bundleId, options.build()};
-        return CommandExecutionHelper.execute(this,
-                new AbstractMap.SimpleEntry<>(TERMINATE_APP, prepareArguments(parameters, values)));
+        Map<String, Object> args = new HashMap<>();
+        args.put("bundleId", bundleId);
+        args.put("appId", bundleId);
+        Optional.ofNullable(options).map(BaseOptions::build).ifPresent(args::putAll);
+        return CommandExecutionHelper.executeScript(this, "mobile: terminateApp", args);
```

Comment:
btw, does this include for mac2 driver as well?  Then, mac2 driver needs `macos` instead of `mobile`
https://github.com/appium/appium-mac2-driver#macos-activateapp

Reply:
no, only AndroidDriver and IOSDriver implement this interface